UnityProject/Assets/Prefabs/Items/Construction/PlaceableTiles/MineralTiles/BananiumTile.prefab

```diff
@@ -45,6 +45,11 @@ PrefabInstance:
       value: 
       objectReference: {fileID: 11400000, guid: 83855008e43a2674d8d6a006ce94326f,
         type: 2}
+    - target: {fileID: 6610653795340409978, guid: b122d05362087574ca3b106604026dd9,
+        type: 3}
+      propertyPath: exportCost
+      value: 250
```

Comment:
this change looks to be unrelated to the PR. At least register the change in the changelog for the players to know it was tweaked.

Reply:
"CL: [Improvement] banana floor tiles actually are worth something 250 credits"